detections/endpoint/windows_admon_default_group_policy_object_modified.yml

```diff
@@ -15,10 +15,10 @@ description: The following analytic detects modifications to the default Group P
   persistence, or deploy malware across multiple hosts. If confirmed malicious, such
   modifications could lead to widespread policy enforcement changes, unauthorized
   access, and potential compromise of the entire domain environment.
-search: ' `admon` admonEventType=Update objectCategory="CN=Group-Policy-Container,CN=Schema,CN=Configuration,DC=*"
-  (displayName="Default Domain Policy" OR displayName="Default Domain Controllers
-  Policy") | stats min(_time) as firstTime max(_time) as lastTime values(gPCFileSysPath)
-  by dcName, displayName | `security_content_ctime(firstTime)` | `security_content_ctime(lastTime)`
+search: ' `admon` admonEventType=Update objectCategory="CN=Group-Policy-Container,CN=Schema,CN=Configuration,DC=*" (displayName="Default Domain Policy" OR displayName="Default Domain Controllers Policy")
+  | appendpipe [
+  | map search="search `wineventlog_security` EventCode=5136  AttributeSyntaxOID=2.5.5.12 AttributeValue=$displayName$" | rename AttributeValue as displayName] 
```

Comment:
@dluxtron  We would need both the datasets here: one for `admon` and the other for 

EventCode=5136. Same with the other admon detection!